### ISSUE TITLE
No files found on macOS

### ISSUE THREAD

**Issue by JonathanMSifleet** [fabiospampinato/vscode-open-multiple-files#13]
Running a 2021 Macbook Pro (M1, Sonoma 14.3.1), and the following glob doesn't work `**/*.tsx`, despite all my files being `.tsx`s
E.g.: `ROOT/src/components/atoms/Button/Button.tsx`
![Untitled 3](https://github.com/fabiospampinato/vscode-open-multiple-files/assets/41328055/ea340627-780f-4073-bbae-6ce00df32c04)

**Comment by fabiospampinato** [comment]
Does it change anything if you put these settings in your settings file?:

```json
  "openMultipleFiles.exclude": [],
  "openMultipleFiles.ignore": []
```

**Comment by JonathanMSifleet** [comment]
I specified macOS as I used to use this on Windows in the past without any issues, however, I tested on Windows and get the same behaviour.

Adding those lines solved it, thanks!

**Comment by fabiospampinato** [comment]
@JonathanMSifleet can we try to narrow down the issue so that I can see if there's something that needs fixing here? For example if you use only either one of those settings, which one of them is responsible for the issue? If setting `openMultipleFiles.exclude` to `[]` fixes it can you post the value of your `files.exclude` setting in vscode? If only `ROOT/src/components/atoms/Button/Button.tsx` fixes it can you post the contents of your `.gitignore` file? There should be something ignoring those `*.tsx` files for some reason, I think 🤔

**Comment by JonathanMSifleet** [comment]
Hi sorry for the delay. Looks like this has been fixed in 2.0.3. Re-open if you feel its necessary and I'll be happy to check it out for you

**Comment by fabiospampinato** [comment]
I just pushed v2.0.4 which may have brought the issue back. If you could give that a try it might be useful. Other than that I'd say if it works for you and you can't spot anything misbehaving we can leave it at that.
